meta-luneos/recipes-core/initscripts/webos-initscripts.bb

```diff
@@ -26,10 +26,10 @@ SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE}"
 SYSTEMD_PACKAGES = "${PN}"
 SYSTEMD_SERVICE_${PN} = "default-webos.target \
                          webos-cbd.target \
-			 webos-ibd.target \
-			 webos-rbd.target \
-		         webos-bd.target \
-			 webos-dis.target \
-			 webos-mbd.target"
+                         webos-ibd.target \
+                         webos-rbd.target \
+                         webos-bd.target \
+                         webos-dis.target \
+                         webos-mbd.target"
```

Comment:
Please use 4 spaces for indentation with closing quite on separate line (as a first character).

Reply:
Updated